pegasus/sites.v3/hourofcode.com/public/css/front-page.scss

```diff
@@ -97,9 +97,14 @@ section.join-us {
   }
 
   .row {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    flex-wrap: wrap;
 
     & > div:first-child {
       margin: 0 auto;
+      float: none;
```

Comment:
All of this moves centers the form when the announcement boxes are hidden for countries outside of the US.